server/utils/db/db_adapters.py

```
@@ -72,7 +72,7 @@ def connect_to_db_as_admin():
     except psycopg2.pool.PoolError:
         raise Exception("connection pool exhausted")
     except Exception as e:
-        logger.error(f"Error getting admin connection: {e}")
+        logger.error("Error getting admin connection: %s", type(e).__name__)
```

Ruff (0.15.12)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)